package.json

```diff
@@ -33,7 +33,8 @@
     "tape": "^4.0.0"
   },
   "dependencies": {
-    "chalk": "^1.1.0",
-    "fastseries": "^1.3.1"
+    "chalk": "2.0.0",
```

Comment:
Can you upgrade this to v3?

Also, depend on it with a ^, so new minor versions will be picked up.

Reply:
I haven't investigated the breaking changes introduced in chalk v3, so I cannot guarantee the current code is compatible with chalk v3.

I do not depend on it with ^, because when running the tests locally with chalk 2.4.0 there are two tests which fail, when running 
	tape test.js | faucet
which is the command run by npm test. However, when just running 
	tape test.js
they succeed. I don't know why this is the case, but do you want the version number to be ^2.0.0 anyway?

Reply:
If we need to update this, I would like it to be updated to the latest version of chalk, v3. This PR should fix the tests as well, thanks.